static/app/views/alerts/create.spec.jsx

```diff
@@ -439,4 +440,51 @@ describe('ProjectAlertsCreate', function () {
       });
     });
   });
+
+  describe('test preview chart', () => {
+    const organization = TestStubs.Organization({features: ['issue-alert-preview']});
+    afterAll(jest.clearAllMocks);
```

Comment:
can we keep this consistent with other tests, it might need to be afterEach
```suggestion
    afterEach(() => {
      jest.clearAllMocks();
    });
```